### DISCUSSION TITLE
Please bring back State of binary Input of HmIPW-DRI16/32

### DISCUSSION THREAD

**Discussion by gamer123** [SukramJ/aiohomematic#1081] [Ideas]
In a earlier version there was the state of HmIPW-DRI16/32 available. 
Now it is just event driven, when button is pressed short or long. 
But this is useless when window contacts are conneceted to detect open and closesd windows. 
Also connected circuitbreaker contacts can't be usesd anymore.

**Comment by danielperna84** [comment]
Please try to delete the device cache and reload the integration, once you have configured the channel to the mode it should operate in.
